mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -579,7 +589,7 @@ class Completions(
                   sym,
                   sym.decodedName,
                   CompletionValue.Workspace(_, _, _, sym),
-                ).forall(visit),
+                ).map(visit).forall(_ == true),
```

Comment:
Is the difference here that it will foreach exec `visit` even if the whole thing returns `false`?

Reply:
Yes, that's it. `forall` would stop at the first that returns false, which wasn't expected.